ops/terraform/services/server/server-regression/docs/server-regression.plantuml

```diff
@@ -0,0 +1,49 @@
+@startuml server-regression
+title server-regression Lambda Deployment and Run Sequence
+participant operator
+database "github"
+participant "bfd-server-regression-build"
+participant "deployment pipeline"
+database "AWS ECR"
+participant "parameter store"
+participant "AWS Lambda"
+participant "sqs queue"
+note over "bfd-server-regression-build","deployment pipeline": distinct Jenkins pipeline jobs ran asynchronously
+operator -> github: Submit change set
+github --> operator
+operator -> "deployment pipeline": Trigger Deployment
+operator -> "bfd-server-regression-build": Trigger Image Build
```

Comment:
As discussed yesterday, I think it's important to illustrate that these are (at least currently) two distinct processes or sequences.